### ISSUE TITLE
The current version isn't working with 1.16.4

### ISSUE THREAD

**Issue by Winddbourne** [bernie-g/geckolib#129]
The current version updates in 1.16.4 but when you try to load it the thing requires a version of forge that you can only pick for a 1.16.5 mod pack in the forge loader. 

[crash-2021-04-22_07.58.03-fml.txt](https://github.com/bernie-g/geckolib/files/6358052/crash-2021-04-22_07.58.03-fml.txt)

As you can see it's asking for forge 36.1.1 but the highest forge level supported by 1.16.4 in the options list is 35.1.37

![version](https://user-images.githubusercontent.com/24589533/115712114-5e49dc80-a342-11eb-9f37-f0fa1199d282.png)

It won't let me choose a higher version number. I considered just updating the pack to 1.16.5 but not all of the mods I'm using are there yet. At the moment I've backed gecko all the way to the last 1.16.4 specific version and will be testing further versions one by until until I get the crash again but . . . I figured you'd want to know the problem exists. 

Having the update arrow stuck there isn't game breaking, but it is annoying when I can't actually update. lol

Edit - I can get it to work without the crash all the way up to 1.16.5-3.0.31,  So it looks like it's only the two versions that came out in this last week that aren't backwards compatible.

**Comment by bernie-g** [comment]
This is on purpose. GeckoLib only supports the latest minor version of each major update.

**Comment by Winddbourne** [comment]
I think our definition of "on purpose" isn't the same . . . When you uploaded your mod you clicked that you were supporting 1.16.4 when you weren't on purpose?  So you actually wanted the app to tell people in an UNSUPPORTED version to update and have their game crash? That's not funny. 

As a mistake it can happen. You click the little check box 1.16.3, 1.16.4 . . . my last update said I was supporting 1.16.5 because I thought the forge versions were backward compatible. I changed that this time around and issued an APOLOGY in my update notes. I made a mistake. No big deal. 

But to say you did it on purpose? I'm not sure if we are communicating well or if your telling me that this was a really weird prank of some kind. If so you got me. I updated the mod trusting the app and through it you, and my game crashed. I even reported the "mistake" so you could fix it.  So funny . . . not really.

**Comment by bernie-g** [comment]
![A9IpOC](https://user-images.githubusercontent.com/24500654/116024097-7c9e2980-a61b-11eb-9430-a288f093bd81.png)

I have no idea what you're talking about. The build we uploaded does not say it supports 1.16.4. We gave people a grace period to update from 1.16.4 to 1.16.5 and that grace period is over.

**Comment by Winddbourne** [comment]
Ok, we are having a communication issue. 

![update](https://user-images.githubusercontent.com/24589533/116109232-99217c80-a682-11eb-9431-6915b6a39907.png)

This is the curse app telling me to update your file in a 1.16.4 modpack. If I do that I'm going to have an immediate crash because the current file doesn't support 1.16.4. 

Normally this happens because you list that you are supporting that version of the latest update when you upload the file. My current pack says it supports 1.16.4 + 1 other version.  Which your screen shot, oddly, doesn't. 

![file](https://user-images.githubusercontent.com/24589533/116109551-e4d42600-a682-11eb-9be3-59c873a16dcf.png)

If you didn't click that you are supporting 1.16.4 I have no idea why Curseforge is telling me that you did. But it's really annoying. I have to remember that this isn't true every time I open up my work space because the app INSISTS that I need to update this file to the latest 1.16.4 supporting version. 

When I clicked the upload button the other day it it actually did upload a new version which crashed my game immediately when I tested it. I also was able to go into files and see two versions that I couldn't use.

Opening it today to get a screen shot the FILE area is suddenly showing I'm on the latest version, but I'm also still getting that update prompt as shown here. 

Perhaps the problem is on Curseforge's end.  It's still annoying though I admit I'm tempted to press the update button again today just to see what it does now. lol
